### ISSUE TITLE
Cannot show custom message in numericality validation

### ISSUE THREAD

**Issue by saobangpro2991** [ansman/validate.js#71]
All custom messages for numericality validation does not work. The messages as below:
```notValid
notInteger
notGreaterThan
notGreaterThanOrEqualTo
notEqualTo
notLessThan
notLessThanOrEqualTo
notOdd
notEven
```
This is my use case:
```
quantity: {
                presence : {
                    message : "^coupon.qty.required"
                },
                numericality: {
                    onlyInteger: true,
                    notInteger: "^coupon.qty.int",
                    greaterThanOrEqualTo: -1,
                    notGreaterThanOrEqualTo: "^coupon.qty.min"
                }
            }
```
When i debugged, i found that there is one issue at line https://github.com/ansman/validate.js/blob/master/validate.js#L834

If we change:
```var msg = this["not" + v.capitalize(name)] || "must be %{type} %{count}";```
to
```var msg = options["not" + v.capitalize(name)] || "must be %{type} %{count}";```
then the custom messages will be shown.

**Comment by ansman** [comment]
This is intentional, there currently isn't any support for setting the messages for individual cases using the options. It's still TODO.

**Comment by ansman** [comment]
This has been fixed in 0.9.0
